### ISSUE TITLE
"There was an error loading this course." while trying example course

### ISSUE THREAD

**Issue by pietrushnic** [overhangio/mu#2]
Steps to reproduce:

```bash
git clone https://github.com/overhangio/mu.git
source ~/venv/bin/activate
cd mu
pip install -e .
cd examples
rm -rf olx*
mkdir olx
mu ./course.md ./olx/
tar czf olx.tar.gz ./olx
```

After the above steps, I'm getting olx.tar.gz, which I'm trying to import into the new course in OpenEdx:

1. In OEX studio -> New Course
2. Fill Create a New Course form:
    * Course Name: My course name
    * Organization: 3mdeb
    * Course Number: Mu1001
    * Course Run: 2023_v1
    * Hit Create
3. Go to Tools-> Import
4. Select olx.tar.gz
5. View the live version of the course - this show "There was an error loading this course."

Is the above procedure correct? If not, what should be changed? If yes, then what can be done to diagnose the problem?

**Comment by regisb** [comment]
Hi @pietrushnic! Can you please share a screenshot of the issue? I am unable to reproduce it on the [demo platform](https://demo.openedx.overhang.io/dashboard), which runs Olive.

It is not clear to me whether the error is occurring in the LMS or the Studio? If the error is happening in the student courseware, you should be able to get more info in the logs of the lms container:

    tutor local logs --tail=100 -f lms

If the error is occurring in the Studio, run instead:

    tutor local logs --tail=100 -f cms

**Comment by Yagnesh1998** [comment]
I have one question I add simply olx.tar.gz to import my course as u provide in the example and I facing this issue here is the logs of cms 
![Screenshot from 2023-04-27 17-29-50](https://user-images.githubusercontent.com/127923546/234856196-af6943b4-0ced-4cf0-a738-e4bf2ddcf2eb.png)
![Screenshot from 2023-04-27 17-30-37](https://user-images.githubusercontent.com/127923546/234856209-45fd567c-3458-4b2b-b32b-d3b236df9ec8.png)

**Comment by Yagnesh1998** [comment]
hello @regisb i have one question i am doing  following steps 
git clone https://github.com/overhangio/mu.git
cd mu
pip install -e .
cd examples
rm -rf olx*
mkdir olx

Markdown -> OLX
mu ./course.md ./olx/

# OLX -> HTML 
mu olx/course/ ./course.html 

after this step I faced the issue is these paths were all correct or if I set the wrong path.

# HTML -> OLX
mu ./course.html ./olx/course/

**Comment by pietrushnic** [comment]
@regisb, sorry for being slow with this. Now I should have more time and focus on the problem.

Please check the screenshots (dashboard):

![oex_dashaboard](https://github.com/overhangio/mu/assets/1071894/b509ac69-a79e-4ff6-b106-03a7bdcbe411)

Course page:

![oex_mu1001_page](https://github.com/overhangio/mu/assets/1071894/5277f106-a9d2-47b9-b6c0-5bee6f6f47bd)

> It is not clear to me whether the error is occurring in the LMS or the Studio?

In the LMS.

Logs from LMS container: http://termbin.com/x634 - there is some Python exception there.

**Comment by Yagnesh1998** [comment]
@pietrushnic I solve this issue index out of range please see my pull request this issue comes in a free text question it mu codes some attributes missing so that is why that happens.

**Comment by regisb** [comment]
Very sorry about the slow turnaround on this! I just published v0.1.2 which should resolve the issue. Can you please confirm @pietrushnic? And thank you very much @Yagnesh1998 for providing a fix!
